easybuild/easyconfigs/m/MCR/MCR-R2017b.eb

```diff
@@ -0,0 +1,23 @@
+name = 'MCR'
+version = 'R2017b'
+
+homepage = 'https://www.mathworks.com/products/compiler/mcr/'
+description = """The MATLAB Runtime is a standalone set of shared libraries
+ that enables the execution of compiled MATLAB applications
+ or components on computers that do not have MATLAB installed."""
+
+toolchain = SYSTEM
+
+source_urls = [
+    'https://www.mathworks.com/supportfiles/downloads/%(version)s/deployment_files/%(version)s/installers/glnxa64/',
+]
+sources = ['%(name)s_%(version)s_glnxa64_installer.zip']
+checksums = ['a01619b30f8cfd34a9a9cefe8323a2ada22a65722fc13b823dae36086ddd259f']
+
+# disable MCR's stock libstdc++.so.6
+# https://www.mathworks.com/matlabcentral/answers/329796-issue-with-libstdc-so-6
+postinstallcmds = [
+    'mv "%(installdir)s/v93/sys/os/glnxa64/libstdc++.so.6" "%(installdir)s/v93/sys/os/glnxa64/libstdc++.so.6-backup"'
```

Comment:
@crubb should we add this to the other MCR versions as well?

Reply:
I would think so, and I think we should drop the Java dependency on the other MCR versions, as well. I actually wanted to see how the discussion turned out on this `.eb`, and then change the other MCR versions, accordingly. ;)

Reply:
@crubb removing the Java dep is fine, as the same was decided for MATLAB-2019a, see #8384

Reply:
Since I'm now running into similar problems with the Qt5 libraries with `ConnectomeWorkbench`, I kept digging around and found some extensive research: https://github.com/mlpack/mlpack/issues/244#issuecomment-68326753 - There simply is no clean one-fits-all-solution, but for the `ConnectomeWorkbench` we need to move the Qt5 libraries out of the way, as well.

Reply:
The MCR dependency was removed from this PR.